pkg/codegen/testing/test/testdata/provider-config-schema/python/pulumi_configstation/provider.py

```diff
@@ -86,7 +106,8 @@ def _internal_init(__self__,
 
             if favorite_color is None:
                 favorite_color = _utilities.get_env('FAVE_COLOR')
-            __props__.__dict__["favorite_color"] = pulumi.Output.from_input(favorite_color).apply(pulumi.runtime.to_json) if favorite_color is not None else None
+            __props__.__dict__["favorite_color"] = pulumi.Output.secret(favorite_color).apply(pulumi.runtime.to_json) if favorite_color is not None else None
+            __props__.__dict__["secret_sandwiches"] = pulumi.Output.from_input(secret_sandwiches).apply(pulumi.runtime.to_json) if secret_sandwiches is not None else None
```

Comment:
Wait, the codegen looks incorrect...

```
__props__.__dict__["favorite_color"] = pulumi.Output.secret(favorite_color)
__props__.__dict__["secret_sandwiches"] = pulumi.Output.from_input(secret_sandwiches)
```

I would expect `favorite_color` to keep using `Output.from_input` and `secret_sandwiches` to use `Output.secret`

Reply:
Oh, I see... In `pkg/codegen/python/gen.go` above, the `if prop.Secret {`  is using `from_input` and the `else` is using `secret` mistakenly. Added a comment on that above.